src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,127 @@
+          massa ultricies iaculis. Quam lacus maecenas nibh malesuada. At
+          tristique et ullamcorper rhoncus amet pharetra aliquet tortor.
+          Suscipit dui, nunc sit dui tellus massa laoreet tellus.
+        </Accordion.Panel>
+      </>
+    ),
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Accordion>;
+
+export const Default: StoryObj<Args> = {};
+
+export const Compact: StoryObj<Args> = {
+  args: {
+    variant: 'compact',
+  },
+};
+
+export const Stacked: StoryObj<Args> = {
+  render: ({ variant, defaultOpen }) => (
+    <div>
+      <Accordion defaultOpen={defaultOpen} variant={variant}>
+        <Accordion.Button>
+          <Accordion.Title as="h2">Massa quam egestas massa.</Accordion.Title>
+        </Accordion.Button>
+        <Accordion.Panel>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla amet,
+          massa ultricies iaculis. Quam lacus maecenas nibh malesuada. At
+          tristique et ullamcorper rhoncus amet pharetra aliquet tortor.
+          Suscipit dui, nunc sit dui tellus massa laoreet tellus.
+        </Accordion.Panel>
+      </Accordion>
+      <Accordion defaultOpen={defaultOpen} variant={variant}>
+        <Accordion.Button>
+          <Accordion.Title as="h2">Massa quam egestas massa.</Accordion.Title>
+        </Accordion.Button>
+        <Accordion.Panel>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla amet,
+          massa ultricies iaculis. Quam lacus maecenas nibh malesuada. At
+          tristique et ullamcorper rhoncus amet pharetra aliquet tortor.
+          Suscipit dui, nunc sit dui tellus massa laoreet tellus.
+        </Accordion.Panel>
+      </Accordion>
+      <Accordion defaultOpen={defaultOpen} variant={variant}>
+        <Accordion.Button>
+          <Accordion.Title as="h2">Massa quam egestas massa.</Accordion.Title>
+        </Accordion.Button>
+        <Accordion.Panel>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla amet,
+          massa ultricies iaculis. Quam lacus maecenas nibh malesuada. At
+          tristique et ullamcorper rhoncus amet pharetra aliquet tortor.
+          Suscipit dui, nunc sit dui tellus massa laoreet tellus.
+        </Accordion.Panel>
+      </Accordion>
+      <Accordion defaultOpen={defaultOpen} variant={variant}>
+        <Accordion.Button>
+          <Accordion.Title as="h2">Massa quam egestas massa.</Accordion.Title>
+        </Accordion.Button>
+        <Accordion.Panel>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla amet,
+          massa ultricies iaculis. Quam lacus maecenas nibh malesuada. At
+          tristique et ullamcorper rhoncus amet pharetra aliquet tortor.
+          Suscipit dui, nunc sit dui tellus massa laoreet tellus.
+        </Accordion.Panel>
+      </Accordion>
+    </div>
+  ),
+};
+
+/**
+ * The following stories are mostly for visual regression testing to capture the open state.
+ */
```

Comment:
Should this comment be after the `StackedCompact` story and before `DefaultOpen` instead?

Reply:
Yes 🤦 thanks for catching